main/getting-started/README.md

```diff
@@ -68,52 +41,77 @@ yarn build
 yarn link-cli /usr/local/bin/agoric
 ```
 
+
 ## Your First Agoric Dapp
 
 ![The Browser Environment](./assets/browser-environment.svg)
 
-### 1. Creating a project directory
-Let's install the default smart contracts and UI.
+In this guide, we're installing, running and interacting with the default smart contracts and UI.
+
+TODO: define/explain the various parts
```

Comment:
The various parts of the diagram are not clear to me. I'm also in doubt with the edges with "HTML"